core/domain/draft_upgrade_services.py

```diff
@@ -206,6 +206,22 @@ def _convert_html_in_draft_change_list(
                 })
         return draft_change_list
 
+    @classmethod
+    def _convert_states_v47_dict_to_v48_dict(cls, draft_change_list):
+        """Converts draft change list from state version 45 to 46. State
+        version 46 ensures that written translations corresponding to
+        unicode text have data_format field set to 'unicode' and that they
+        do not contain any HTML tags. This should not affect drafts.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        return draft_change_list
```

Comment:
Are you sure existing drafts will not be affected by the change you are making? I would have thought you'd need to go through them using _convert_html_in_draft_change_list() and update them appropriately, similar to what was in the v46-to-v47 conversion before.